kd/autoscale.yaml

```
@@ -5,7 +5,7 @@ metadata:
     app: hocs-info-service
   name: hocs-info-service
 spec:
-  maxReplicas: 2
+  maxReplicas: {{.MAX_REPLICAS}}
   minReplicas: {{.REPLICAS}}
```

May as well name them consistently?
```suggestion
  minReplicas: {{.MIN_REPLICAS}}
```